views/forms/error-summary/code.html.twig

```diff
@@ -0,0 +1,135 @@
+{% extends '@pulsar/pulsar/components/tab.html.twig' %}
+{% import 'helpers/example.html.twig' as docs %}
+
+{% block tab_content %}
+<div class="report">
+    <p class="lede">Display a summary of form errors. The summary links to each input with an error.</p>
+
+    {{ docs.heading('h2', 'Basic usage') }}
+
+    <p>The error summary helper accepts an array of objects. Each object contains the error text and  ID of an input with errors. Refer to the <a href="/guides/error-messages#form-errors">Error Summary guide</a> for more information on usage.</p>
+
+    {{ docs.example({
+        screenreader: "",
```

Comment:
It would be good to highlight what a screenreader will announce for this component